gammapy/datasets/core.py

```diff
@@ -505,7 +505,8 @@ def info_table(self, cumulative=False):
         Parameters
         ----------
         cumulative : bool
-            Cumulate info across all observations. Default is False.
+            Cumulate information across all datasets. If True, all information depending
+            on a model will be lost. Default is False.
```

Comment:
`If True, all information depending on a model will be lost.`

I'm still not sure what is meant by this, can you clarify?
`If True, all information associated with the model will be lost.` ??

Reply:
It means that because the models are not copied when you stack datasets, informations that depends on model are lost. This is for instance `stat_sum`, or `npred_signal` in a `MapDataset` or `SpectrumDataset`. So this part of the docstring is here to notify the user that it is normal behaviour to get a `NaN` in the `stat_sum` column in these cases. 
If you have a better formulation for this @Astro-Kirsty, I will be please to changed it.

Reply:
Okay, your comment makes sense but I would suggest to change the docstring slightly. Even as you described in your comment:
"If True, all information that depends on the model are lost"
OR
"If True, all model-dependent information will be lost."
Or another option, if you have it. But maybe avoid using the wording 'depending'